applications/bci_visualization/operators/voxel_stream_to_volume/voxel_stream_to_volume.py

```
@@ -0,0 +1,292 @@
+        spec.output("mask_spacing").condition(ConditionType.NONE)
+        spec.output("mask_permute_axis").condition(ConditionType.NONE)
+        spec.output("mask_flip_axes").condition(ConditionType.NONE)
+
+    def compute(self, op_input, op_output, context):
+        # Receive Hb voxel data (cupy array)
+        hb_voxel = op_input.receive("hb_voxel_data")  # (I, J, K)
+        cuda_stream = op_input.receive_cuda_stream("hb_voxel_data")
+
+        # Check voxel data is valid
+        if not isinstance(hb_voxel, cp.ndarray):
+            raise ValueError("VoxelStreamToVolume: Invalid voxel data, expected cupy array")
```

Ruff (0.14.10)

Prefer `TypeError` exception for invalid type

(TRY004)